tests/reg_tests/test_DVConstraints.py

```diff
@@ -732,6 +778,34 @@ def test_monotonic(self, train=False, refDeriv=False):
                 atol=1e-7,
             )
 
+    @unittest.skipIf(missing_geograd, "requires geograd")
```

Comment:
So, the other test is on its own only because cannot be used under parameterized?

Reply:
Yes, that's right